samples/word/07-custom-properties/get-built-in-properties.yaml

```diff
@@ -0,0 +1,96 @@
+id: word-basics-get-built-in-properties-
+name: Get built-in document properties.
+description: Shows how to get built-in document properties.
+author: Microsoft
+host: WORD
+api_set:
+    WordApi: 1.1
+script:
+    content: |+
+        $("#run").click(getProperties);
+
+        function getProperties() {
+            Word.run(function (context) {
+                var builtInProperties = context.document.properties;
+                context.load(builtInProperties);
+                return context.sync()
+                    .then(function () {
```

Comment:
You could just do JSON.stringify(builtInProperties, null, 4) and get them all at once...